infrastructure/template.yaml

```diff
@@ -16,6 +16,22 @@ Parameters:
       - "integration"
       - "production"
     ConstraintDescription: must be dev, build, staging, integration or production
+  SlackWorkspaceId:
+    Description: >
+      The ID of the Slack workspace where build notification messages are
+      posted. This is retrieved from the AWS Chatbot integration.
+    Type: String
+    Default: "T8GT9416G"
```

Comment:
I take it this is the ID for the GDS workspace (gds.slack.com?)? And then `SlackChannelId` is our team tech channel?

Reply:
Yes that's right. I got these values from the dev platform pipeline stack in the `build` AWS account

Reply:
In fact, this whole parameter definition is taken directly from that stack